clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -3651,6 +3674,21 @@ static FormatToken *getFunctionName(const AnnotatedLine &Line,
       continue;
     }
 
+    // Skip past template typename declarations that may precede the
+    // constructor/destructor name
```

Comment:
```suggestion
    // Skip past template typename declarations that may precede the
    // constructor/destructor name.
```
Comments end in full stop.

Reply:
Done.